packages/editor/src/editor/properties/fields/table-field/SelectTableField.tsx

```diff
@@ -0,0 +1,26 @@
+import { type ColumnDef } from '@tanstack/react-table';
+import type { SelectItem } from '@axonivy/form-editor-protocol';
+import { InputCellWithBrowser } from './InputCellWithBrowser';
+import { TableField, type TableFieldProps } from './TableField';
+
+type SelectTableFieldProps = Omit<TableFieldProps<SelectItem>, 'columns' | 'emptyDataObject'>;
+
+const selectTableColumns: ColumnDef<SelectItem, string>[] = [
```

Comment:
I had to define the SelectTableColumns outside the SelectTableField component, otherwise the table behaved strangely (values inserted by the browser were not saved or navigation via tab did not work). But no idea why.